render/pixel.go

```diff
@@ -65,3 +68,11 @@ func (c BGR555) MarshalJSON() ([]byte, error) {
 		"a": a,
 	})
 }
+
+var BGR555Model = color.ModelFunc(func(c color.Color) color.Color {
```

Comment:
nit: can we add this as a screen attribute instead of a global variable ?

Reply:
This is not tied to the screen but to the color type. This is how it is done in the stdlib: https://golang.org/pkg/image/color/#Model

Reply:
:+1: but it's only used in the screen method so we could just create it when creating a new screen no ?

Reply:
done